yaml/landuse.yaml

```diff
@@ -323,9 +326,6 @@ filters:
   - filter: {tourism: picnic_site}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
     output: {kind: picnic_site}
-  - filter: {historic: fort}
-    min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: fort}
```

Comment:
I recall needing to move `fort` above attraction a few weeks ago as they are sometimes also tagged `attraction`. So moving it is "okay", but confusing why it wasn't already up there.